students/WesleyM/Lesson4/trigrams.py

```diff
@@ -0,0 +1,42 @@
+import re
+import random
+
+def open_file(string):
+    '''Opens, reads and closes a .txt file'''
+    f = open(string)
+    secret_data = f.read()
+    f.close()
+    return secret_data
+
+def ngrams(input, n):
+    '''Takes text and makes n-grams'''
+    input = input.split(' ')
```

Comment:
It's a good idea to avoid using built-in names like `input` and `dict` for variables. One convention in Python is to use a trailing underscore like `input_`, or use a more descriptive name like `trigram_dict`